duecredit/tests/test__main__.py

```diff
@@ -8,15 +8,18 @@
 # ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
 
 import sys
-import pytest
-
 from io import StringIO
+from typing import TYPE_CHECKING
+
+import pytest
 
 from .. import __main__, __version__
 from .. import due
 
+if TYPE_CHECKING:
```

Comment:
Why guard this behind `if TYPE_CHECKING`?  Unless you're supporting pre-7.0 pytest (Why would you?), you can just import these types unconditionally.